OBIII.service

```diff
@@ -4,6 +4,7 @@ After=multi-user.target
 
 [Service]
 Type=simple
+User=root
```

Comment:
Require for OBIII to use GOPATH properly

Reply:
I agree, lets not introduce issues with plans to fix them later. let's just do it right the first time

Reply:
I was under the assumption that OBIII ran as root, so I didn't see this as an issue. I am realizing because I didn't specify before it was likely running under systemd. Nonetheless, OBIII working directory is /root/ops-bot-iii, #28 will deal with that issue.

I will go back and try to get everything working without setting root as the default user.

Reply:
Fixed in cefcb7996f0587308d8e2bd41ae5b0b3e2352b9a)